resolwe_bio/tools/basespace_download.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+"""Tool to download files from BaseSpace."""
+
+import sys
+import traceback
+import atexit
+import argparse
+import requests
+
+
+class BaseSpaceDownloadError(Exception):
+    """BaseSpace download error."""
+
+    pass
+
+
+def main():
+    """Entry point."""
+    session = requests.Session()
+    atexit.register(on_exit, session)
+
+    parser = argparse.ArgumentParser(description='Download file from Illumina BaseSpace.')
+    parser.add_argument('--file-id',
+                        dest='file_ids',
+                        action='append',
+                        required=True,
+                        help='BaseSpace file ID. This argument can be repeated to specify multiple files.')
+    parser.add_argument('--access-token-secret-path',
+                        dest='access_token_secret_path',
+                        required=True,
+                        help='BaseSpace access token secret path.')
+    parser.add_argument('--verbose',
```

Comment:
@mzganec `basespace_download` tool accepts `--verbose` flag. Would it make sense to add an input to [basespace-import-file](https://github.com/genialis/resolwe-bio/pull/479/files#diff-9d218a7a04cd5acd878972b5e7ece1cfR18) process?

Reply:
We usually don't do this.

Reply:
The reason for this would be to able to debug in case of errors.

Reply:
I understand. You can certainly include this option but put it in the _Advanced_ group and hide it by default (see [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/import_data/sra_file.yml#L30)) because this option is not for basic users.

Reply:
Also verify how much output is being produced. We've had cases with more than a GB of standard output produced in verbose mode.

Reply:
>You can certainly include this option but put it in the Advanced group and hide it by default (see here) because this option is not for basic users.

Agree, but also be aware this process won't be used by a user directly.

Reply:
>Also verify how much output is being produced. We've had cases with more than a GB of standard output produced in verbose mode.

It's just a stack trace.